### ISSUE TITLE
Why bower_components folder is included?

### ISSUE THREAD

**Issue by adrigm** [ismaelfaro/Web-APP-Polymer-Base#2]
It is not enough to upload the file bower.json?

**Comment by abdonrd** [comment]
+1

**Comment by ismaelfaro** [comment]
Yes, I agree. I based it in the initial versions of Google Polymer StarterKit, https://github.com/polymerelements/polymer-starter-kit/releases and they put all web components.

I go to change it.
